app/src/main/java/org/jellyfin/androidtv/browsing/HomeFragment.java

```diff
@@ -179,6 +179,7 @@ public void onResponse(ItemsResult response) {
                         case TvShows:
                             addNextUp();
                             addPremieres();
+                            addLatestTVShows();
```

Comment:
I would move this after Latest Movies to be consistent with the other cases.

Reply:
Okay, will do.